components/template-mgt/org.wso2.carbon.identity.template.mgt/src/main/java/org/wso2/carbon/identity/template/mgt/TemplateMgtConstants.java

```diff
@@ -41,6 +46,30 @@
     public static final String STATUS_INTERNAL_SERVER_ERROR_MESSAGE_DEFAULT = "Internal server error";
 
     public static final String TEMPLATE_RESOURCE_PATH = "/";
+    public static final Path SP_TEMPLATES_DIR_PATH = Paths.get(CarbonUtils.getCarbonHome(), "repository",
+            "resources", "identity", "application-templates");
+    public static final Path IDP_TEMPLATES_DIR_PATH = Paths.get(CarbonUtils.getCarbonHome(), "repository",
+            "resources", "identity", "identity-provider-templates");
+    public static final String FILE_EXT_JSON = ".json";
+
+    // Common attributes to all templates
```

Comment:
add fullstop at the end of a comment. Please check other places as well

Reply:
addressed in feace15